composer.json

```diff
@@ -16,8 +16,16 @@
 		"behat/behat": "^3.7|^3.8",
 		"yoast/wp-test-utils": "^1.0.0",
 		"rector/rector": "^1.1",
-		"wpsyntex/wp-phpunit": "dev-master"
+		"wpsyntex/wp-phpunit": "dev-master",
+		"wpsyntex/object-cache-annihilator": "dev-master"
 	},
+	"repositories": [
+		{
+			"name": "wpsyntex/object-cache-annihilator",
+			"type": "vcs",
+			"url":  "git@github.com:polylang/object-cache-annihilator.git"
+		}
+	],
```

Comment:
I've added the package to Packagist: https://packagist.org/packages/wpsyntex/object-cache-annihilator
SO you shouldn't need this.

Reply:
Nice 👍 
Done in 1fd4beb0.